src/chain/chain_information.rs

```diff
@@ -243,12 +246,12 @@ pub enum ChainInformationConsensus {
         /// Babe epoch information about the epoch right after the one the finalized block belongs
         /// to.
         ///
-        /// If [`ChainInformationConsensus::Babe::finalized_block_epoch_information`] is `Some`,
+        /// If [`ChainInformationConsensus::babe_config::finalized_block_epoch_information`] is `Some`,
```

Comment:
```suggestion
        /// If [`ChainInformationConsensus::Babe::finalized_block_epoch_information`] is `Some`,
```

That's actually not supposed to pass CI!